mlrun/api/api/utils.py

```diff
@@ -168,8 +174,23 @@ def _generate_function_and_task_from_submit_run_body(db_session: Session, data):
         function = new_function(runtime=function_dict)
     else:
         if "://" in function_url:
-            function = import_function(
-                url=function_url, project=task.get("metadata", {}).get("project")
+            project_name = task.get("metadata", {}).get("project")
+            hub_function = function_url.startswith(mlrun.utils.hub_prefix)
+            db = mlrun.api.api.utils.get_run_db_instance(db_session)
+            if function_url.startswith(mlrun.utils.db_prefix):
+                url = function_url.removeprefix(mlrun.utils.db_prefix)
+                _project_name, name, tag, hash_key = parse_versioned_object_uri(url)
+                runtime = db.get_function(name, _project_name, tag, hash_key)
+                if not runtime:
+                    raise mlrun.errors.MLRunNotFoundError(
+                        f"function {name}:{tag} not found in the DB",
```

Comment:
I think we should make it a convention to use capital letters in logs and exceptions
```suggestion
                        f"Function {name}:{tag} not found in the DB",
```